### DISCUSSION TITLE
我Halo后台在页面删除了/about这个页面，怎么重新配置主题里的“关于”并显示呢

### DISCUSSION THREAD

**Discussion by FunkyGod** [liuzhihang/halo-theme-hao#689] [Q&A]
### 是什么版本出现了此问题？

1.4.7

### 在线站点地址

https://funkygod.vip

### 发生了什么？

![image](https://github.com/liuzhihang/halo-theme-hao/assets/39259580/920f3afa-02d9-4d3c-85e4-137af6aeb593)

![image](https://github.com/liuzhihang/halo-theme-hao/assets/39259580/880d8ad3-c020-4a27-9d42-1ca52d924c62)

**Comment by carolcoral** [comment]
https://www.yuque.com/liuzhihangs/halo-theme-hao/eeoayhggq5a7wh3u
